lib/tasks/solid_errors_tasks.rake

```diff
@@ -0,0 +1,17 @@
+namespace :solid_errors do
+  desc "Copy and run migrations from Solid Errors"
+  task install_migrations: :environment do
+    source = File.expand_path("../../solid_errors/db/migrate", __FILE__)
+    destination = Rails.root.join('db', 'migrate')
+
+    Dir.children(source).each do |migration|
+      timestamp = Time.now.strftime('%Y%m%d%H%M%S')
+      filename_with_timestamp = "#{timestamp}_#{migration}"
+
+      if Dir.glob("#{destination}/*#{original_filename}").empty?
+        puts "Copying #{migration} to #{filename_with_timestamp} from Solid Errors to Rails application"
+        FileUtils.cp(migration, File.join(destination, filename_with_timestamp))
+      end
+    end
+  end
+end
```

Comment:
fyi, Rails Engines have this functionality built in. I don't use it in GoodJob, but that was more out of ignorance at the time than anything else.

https://guides.rubyonrails.org/engines.html#engine-setup

Solid Queue does it really well:

https://github.com/rails/solid_queue/blob/ac477d7b45f530026f36456ee114f223d86fd4ef/lib/generators/solid_queue/install/install_generator.rb

Reply:
This is really great! Thanks so much @bensheldon (particularly for the great write-up above!)